### PR TITLE
NEWS: Getting news ready for v1.6.x

### PR DESCRIPTION
**Pull request by shamisp**
Use the opportunity to reflect v1.5.1 in master news file.

Signed-off-by: Pavel Shamis (Pasha) <pasharesearch@gmail.com>

### CODE REVIEW

**Comment by shamisp** [comment]
Once it is in master I will port to v1.6.x tree

File: NEWS [hunk 1]
```diff
@@ -11,12 +11,36 @@
...
+- Modular architecture for UCT transports
+- ROCM transport re-design: support for managed memory, direct copy, ROCM GDR
+- Random scheduling policy for DC transport
+- Optimized out-of-box settings for multi-rail
+- Added support for OmniPath (using Verbs)
+- Support for PCI atomics with IB transports
+- Reduced UCP address size for homogeneous environments
+
+Bugfixes:
+- Multiple stability and performance improvements in  TCP transport
```
Comment (dmitrygladkov) [inline NEWS]: excess space between `in` and `TCP`

File: NEWS [hunk 2]
```diff
@@ -11,12 +11,36 @@
 Features:
 - TBD
 
+## 1.6.0-rc1 (May 24, 2019)
+Features:
+- Modular architecture for UCT transports
+- ROCM transport re-design: support for managed memory, direct copy, ROCM GDR
```
Comment (yosefe) [inline NEWS]: ROCM -> ROCm

File: NEWS [hunk 3]
```diff
@@ -11,12 +11,36 @@
...
+- Multiple stability fixed in Verbs and MLX5 transports
+- Multiple stability fixes in UCM memory hooks
+- Multiple stability fixes in UGNI transport
+- RPM Spec file cleanup
+- Fixing compilation issues with most recent clang and gcc compilers
+
 Tested configurations:
 - RDMA: MLNX_OFED 4.5, distribution inbox drivers, rdma-core 22.1
-- CUDA: gdrcopy 1.3.2, cuda 9.2
+- CUDA: gdrcopy 1.3.2, cuda 9.2, ROCM 2.1
```
Comment (yosefe) [inline NEWS]: ROCM -> ROCm
@yuq @bbenton is this the right version?

Comment (yuq) [reply NEWS]: ROCm 2.1 has a bug which can't support >4G buffer transfer, ROCm >=2.2 is fine.

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10366/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/10371/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
bot:bgate:retest

**Pull request merged**